framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/WorkbenchRuntimeDeploymentImpl.java

```diff
@@ -99,4 +99,14 @@ public void setServiceName(String applicationName) {
             getSecureUrl().ifPresent(RouterUtil::waitForRouter);
         }
     }
+
+    @Override
+    public void changePassword(String newPassword) {
+        throw new UnsupportedOperationException("Not supported");
+    }
+
+    @Override
+    public void changeUsernameAndPassword(String newUsername, String newPassword) {
+        throw new UnsupportedOperationException("Not supported");
+    }
```

Comment:
I think this is a leftover. If we remove the changes in HttpsWorkbenchTestProvider, this is not used anywhere.